pyof/v0x01/symmetric/vendor_header.py

```diff
@@ -21,15 +21,19 @@ class VendorHeader(GenericMessage):
 
     header = Header(message_type=Type.OFPT_VENDOR)
     vendor = UBInt32()
+    additional_data = BinaryData()
 
-    def __init__(self, xid=None, vendor=None):
+    def __init__(self, xid=None, vendor=None, additional_data=b''):
```

Comment:
why `b=''` and not `None`?

Reply:
OK, I will change this.

Reply:
This is how it has been done in the project so far for all BinaryData fields. We need todiscuss and determine better defaults and change the init methods later.

Reply:
But @macartur has just made a PR to change it....

Reply:
Touché; my bad.